docs/source/enterprise-guide/change-management.mdx

```diff
@@ -0,0 +1,543 @@
+> hand, if we have a continuous delivery release process and the time between
+> merge and release is relatively short, it's less likely for teams to introduce
+> incompatible changes with unreleased subgraph schemas.
+
+### Rollbacks
+
+Once clients start using the new schema changes, rolling back a subgraph schema
+change is the same as making a backward incompatible schema change. This may be
+difficult or impossible to do depending on client usage. For this reason, it's
+important to test our new schema changes thoroughly before clients start using
+the new schema in their operations.
+
+If clients are not yet using the schema changes, we most likely don't need to
+roll them back — they have no effect until operations start using them.
+
+If circumstances require a rollback, the safest process is to revert the change
+in the subgraph code repository and perform the full release process, publishing
+the subgraph schema and rolling out the new code together.
+
+### Staged schema change releases
+
+Instead of attempting rollbacks of schema changes, we may want to release our
+changes in stages, such as alpha, beta, and generally-available (GA). The
+upcoming Studio feature "Contracts" makes this process simple:
+
+1. "Tag" new elements of our schema with labels for the targeted release:
+
+   ```graphql
+   type Product {
+     id: ID!
+     newField: String @tag(name: "alpha")
+   }
+   ```
+
+2. Configure our GA variant to be a "contract" that filters out all fields
+   tagged "alpha" or "beta".
+3. Configure a separate beta variant to exclude the "alpha" tag, and another
+   alpha variant that doesn't exclude anything.
+4. Set up three separate gateways configured to use the alpha, beta, and GA
+   variants.
+5. Perform the standard release process. Because we've tagged our new field with
+   "alpha", the schema for the beta and GA variants will not change. This gives
+   us plenty of opportunity to test our change with alpha versions of our
+   clients.
+6. Releasing the change to a wider audience is simply a matter of changing the
+   tag to "beta" or removing it altogether.
+
+## Backward incompatible subgraph schema changes
+
+Backward incompatible schema changes remove or alter schema elements in ways
+that would break existing operations.
+
+Crucially, if a schema element is unused, this does _not_ qualify as a backward
+incompatible change.
+
+### Release steps
+
+The release steps for backward incompatible changes are similar to those for
+backward compatible changes, only with some important pre-work:
+
+1. Use the `@deprecated` directive to communicate with clients that schema
+   elements should no longer be relied on.
+2. Use Apollo Studio to track the usage of types and fields. Reach out to
+   clients [identified][clientids] as performing the affected operations before
+   releasing the change.
+3. Continue to track usage until it reaches zero (or close enough).
```

Comment:
Should we unpack this further, maybe with an example?

(or to a minimally acceptable level, for example, where a field is still in use by an older version of a mobile application that is no longer supported)

Just a thought... feel free to play with the wording too.